services/QuillLMS/app/controllers/admins_controller.rb

```diff
@@ -147,8 +147,9 @@ def create_and_link_accounts
     if SchoolsAdmins.exists?(user: @teacher, school: @school)
       @message = t('admin_created_account.existing_account.admin.linked', school_name: @school.name)
     else
+      existing_admin = SchoolsAdmins.find_by(user_id: @teacher.id)
```

Comment:
I would use a `exists?` here since we aren't using the record at all, but merely just it's existence:

`SchoolsAdmins.exists?(user_id: @teacher.id)`

The `find_by` usage [here](https://github.com/empirical-org/Empirical-Core/blob/d99add1f70ccf26434fe0a0410fc745ab37779bb/services/QuillLMS/app/controllers/cms/school_admins_controller.rb#L63) is appropriate since we're using the record's values.